tensorflow_serving/batching/basic_batch_scheduler.h

```diff
@@ -155,15 +155,14 @@ class BasicBatchScheduler : public BatchScheduler<TaskType> {
 
     // The number of threads to use to process batches.
     //
-    // This parameter also determines the maximum number of batches worth of
-    // tasks that can be enqueued: If all batch threads are busy, Schedule()
-    // will accept at most 'num_batch_threads' additional batches worth of tasks
-    // before rejecting tasks with an UNAVAILABLE error.
+    // If all threads are busy, Schedule() will enqueue up to 
+    // max_enqueued_batches * max_batch_size individual tasks before throwing
```

Comment:
Will do! I didn't realize that a `task` does not necessarily correspond one-to-one to a datum or operation. I'll make that change. Note that this was the source of my confusion in #10.